db/schema.rb

```diff
@@ -171,7 +171,7 @@
 
   create_table "interview_surveys", force: :cascade do |t|
     t.bigint "user_id", null: false
-    t.date "interview_date"
+    t.date "interview_date", null: false
```

Comment:
Nit: looks like the migration might have accidentally been left out of the commit.